src/hooks/useChromeLocalStorage.js

```diff
@@ -15,13 +15,18 @@ export function useChromeLocalStorage(key) {
     const readLocalStorage = async () => {
       try {
         const value = await readFromLocalStorage(key)
-        setData(value[key])
+        if (
+          value[key] &&
```

Comment:
Can `readFromLocalStorage` return an undefined?
Additionally, what happens when the storage is empty, can it be updated for the first time?

Reply:
No, if the key/keys are not there it returns an empty object. Do you think it's necessary to write in the storage the default value?

Reply:
Yes, because data is initialised with the default. I don't want to update it with undefined if it is not in the storage